### DISCUSSION TITLE
Un-vote on polls

### DISCUSSION THREAD

**Discussion by squidink7** [element-hq/element-meta#696] [Ideas]
#### What would you like to do?
Add the ability to redact a vote and go back to being neutral, by deselecting the vote option

#### Why would you like to do it?
Sometimes a user would like to change their position on a poll, and theres already a method for changing your vote, but none for removing your vote altogether. Sometimes simple yes/no polls don't allow for a neutral vote and being able to cancel would be nice.

#### How would you like to achieve it?
Clicking the currently voted for item again could undo the vote and return to the original state

### Have you considered any alternatives?

This could lead to biased answers from people looking at others votes, however changing to another option is already allowed, which also can lead to biases and it hasn't presented to be an issue (at least in the rooms I'm in).

### Additional context

This would defeat the purpose of hiding results until a vote is cast, however some people are already proposing showing results without having to vote, so if that gets accepted it would be appreciated if you would consider implementing something similar to this too.

And thank you for spending time working on this awesome service!

**Comment by kittykat** [comment]
Moving to discussions as this is cross-platform :+1:
